application/src/main/java/org/togetherjava/tjbot/commands/Commands.java

```diff
@@ -70,6 +71,10 @@
         commands.add(new MuteCommand(actionsStore));
         commands.add(new UnmuteCommand(actionsStore));
 
+        ModmailCommand modmailCommand = new ModmailCommand();
+        commands.add(modmailCommand);
+        commands.add(modmailCommand.new ReloadModmailCommand());
```

Comment:
This will not compile.

Reply:
Really? I'm sure I've tested and ran it on my machine numerous times.

Reply:
I also ran gradle `build` so I'm really sure it works. Do you mind checking it again?

Reply:
It doesn't compile no, I'm not sure what you're doing there `modmailCommand.new ReloadModmailCommand()`

Reply:
The `ReloadModmailCommand` is an inner class since it's the only solution I could think of to reload the mods in the `/modmail` command. Thus, I am [instantiating](https://docs.oracle.com/javase/tutorial/java/javaOO/nested.html) the `ReloadModmailCommand` from the `Modmail` object.

Reply:
Ah that explains

Reply:
I am genuinely confused why it doesn't compile. Do you mind helping me out on this one?

Reply:
Well, it compiles like you said, it's just done in a reallyyyyy weird way.
To my knowledge, you can remove `modmailCommand.` from `modmailCommand.new ReloadModmailCommand()`
So please do that, if you can't, refer to the class instead of an instance?

Reply:
You can't. That's the required syntax for instantiating a non-static inner class. It's really weird.